modules/system/chat-wfrp4e.js

```diff
@@ -443,12 +452,12 @@ export default class ChatWFRP {
       return ui.notifications.error(game.i18n.localize("CONDITION.ApplyError"))
 
     if (game.user.isGM)
-      message.update(conditionResult)
+      await message.update(conditionResult)
     else
-      game.socket.emit("system.wfrp4e", { type: "updateMsg", payload: { id: msgId, updateData: conditionResult } })
+      await WFRP_Utility.awaitSocket(game.user, "updateMsg", { id: msgId, updateData: conditionResult }, "executing condition script");
   }
```

Comment:
Don't see the point of awaiting these

Reply:
probably could be removed, i left it for consistency with other method executions